docs/releasing.md

```diff
@@ -1,116 +1,141 @@
-# Releasing
+# cluster-api-provider-metal3 releasing
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+This document details the steps to create a release for
+`cluster-api-provider-metal3` aka CAPM3.
 
-- [Prerequisites](#prerequisites)
-   - [`docker`](#docker)
-- [Output](#output)
-  - [Expected artifacts](#expected-artifacts)
-  - [Artifact locations](#artifact-locations)
-- [Process](#process)
-  - [Permissions](#permissions)
+## Before making a release
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+Things you should do before making a release:
 
-## Prerequisites
+- Uplift controller to use latest corresponding CAPI modules
+- Uplift to latest corresponding IPAM module
+- Uplift to latest corresponding BMO module
+- Uplift any other direct/indirect dependency to close any public
+  vulnerabilities
 
-### `docker`
+## Permissions
 
-You must have docker installed.
+Creating a release requires repository `write` permissions for:
 
-## Output
+- Tag pushing
+- Branch creation
+- GitHub Release publishing
 
-### Expected artifacts
+These permissions are implicit for the org admins and repository admins.
+Release team gets their permissions via `metal3-release-team` membership. Adding
+person to the team gives him/her the necessary rights in all relevant
+repositories in the organization. Individual persons should not be added
```

Comment:
Individual persons should not be given the permissions directly.

Reply:
Fixed.